packages/react-components/react-carousel/library/src/components/useEmblaCarousel.ts

```diff
@@ -114,6 +116,27 @@ export function useEmblaCarousel(
     return plugins;
   }, [motion, onDragEvent, watchDrag]);
 
+  /* This function enables autoplay to pause/play without affecting underlying state
+   * Useful for pausing on focus etc. without having to reinitialize or set autoplay to off
+   */
+  const enableAutoplay = React.useCallback(
+    (_autoplay: boolean, temporary?: boolean) => {
+      if (!temporary) {
+        autoplayRef.current = _autoplay;
+      }
+
+      if (_autoplay && autoplayRef.current) {
+        // Autoplay should only enable in the case where underlying state is true, temporary should not override
+        emblaApi.current?.plugins().autoplay?.play();
+        // Reset after play to ensure timing and any focus/mouse pause state is reset.
+        resetAutoplay();
+      } else if (!_autoplay) {
+        emblaApi.current?.plugins().autoplay?.stop();
+      }
+    },
+    [resetAutoplay],
+  );
```

Comment:
```suggestion
  const enableAutoplay = React.useCallback(
    (autoplay: boolean, temporary?: boolean) => {
      if (!temporary) {
        autoplayRef.current = autoplay;
      }

      if (autoplay && autoplayRef.current) {
        // Autoplay should only enable in the case where underlying state is true, temporary should not override
        emblaApi.current?.plugins().autoplay?.play();
        // Reset after play to ensure timing and any focus/mouse pause state is reset.
        resetAutoplay();
      } else if (!autoplay) {
        emblaApi.current?.plugins().autoplay?.stop();
      }
    },
    [resetAutoplay],
  );
```

nit: we don't need `_` after the latest changes 🐱